src/lang/en-US.json

```diff
@@ -450,6 +452,8 @@
   "hero_2000_win_rate": "2K W%",
   "hero_1000_pick_rate": "<2K P%",
   "hero_1000_win_rate": "<2K W%",
+  "hero_turbo_pick_rate": "Turbo P%",
```

Comment:
should probably expand to Pick and Win since we have the room

Reply:
sounds good, will update